Mathlib/AlgebraicTopology/ModelCategory/IsCofibrant.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.AlgebraicTopology.ModelCategory.CategoryWithCofibrations
+import Mathlib.CategoryTheory.Limits.Shapes.Terminal
+
+/-!
+# Fibrant and cofibrant objects in a model category
+
+Once a category `C` has been endowed with a `CategoryWithCofibrations C`
+instance, it is possible to define the property `IsCofibrant X` for
+any `X : C` as an abbreviation for `Cofibration (initial.to X : ⊥_ C ⟶ X)`.
```

Comment:
Not sure if you meant to also mention the `IsFibrant X` case here in the doc string, but in any case, I think it is quite clear with or without it.

Reply:
I have added a remark about fibrant objects in the docstring.